cached_logos/cilium.svg

```diff
@@ -1 +1,5 @@
+>>>>>>> origin/main
```

Comment:
Beware, this file and a few others contain the merge conflict markers.

Reply:
@mohammed90 fixed!